src/components/core/DpTooltipIcon.vue

```diff
@@ -6,6 +6,8 @@
 </template>
 
 <script>
+import Tooltip from './../../directives/Tooltip/Tooltip'
```

Comment:
```suggestion
import { Tooltip } from '../../directives'
```

And don't we have to add
`tooltip: Tooltip`
to `directives`?